app/controllers/users_controller.rb

```diff
@@ -6,6 +6,42 @@ def index
     @users = @q.result(distinct: true)
   end
 
+  def new
+    authorize User
+    @user = User.new
+  end
+
+  def create
+    authorize User
+    @user = User.new(user_params)
+    @user.password = 'secret'
+    @user.admin = false
+    if @user.save
+      redirect_to admins_path
+      flash[:success] = 'User created'
+    else
+      flash.now[:danger] = 'Correct the field'
```

Comment:
That's quite vague, I think that it would be enough to use something like "There were same errors when saving the user"